script.js

```diff
@@ -32,62 +32,86 @@ pageBody.appendChild(mainPage);
 const bookList = document.getElementById('book-list');
 const bookForm = document.getElementById('add-book');
 
-let books = [];
+class BookArray {
+  constructor() {
+    this.books = [];
+  }
+
+  createBook(title, author) {
+    const book = {
+      title,
+      author,
+    };
+    this.books.push(book);
+  }
+
+  removeBookFromArray(bookNum) {
+    this.books.splice(bookNum, 1);
+  }
+}
 
-function displayBooks() {
-  books.forEach((book, bookNum) => {
-    const bookDiv = document.createElement('div');
-    bookDiv.classList.add(`${bookNum}`);
+const booksObj = new BookArray();
+// eslint-disable-next-line prefer-destructuring
+let books = booksObj.books;
 
-    bookDiv.innerHTML = `
-      <h2></h2>
-      <h3></h3>
-      <button type="button" onmousedown="removeBook(${bookNum})">Remove</button>`;
+function addBookToPage(title, author, bookNum) {
+  const bookDiv = document.createElement('div');
+  const bookId = bookNum;
+  bookDiv.classList.add(`book-${bookId}`);
 
-    bookDiv.children[0].textContent = `${book.title}`;
-    bookDiv.children[1].textContent = `${book.author}`;
+  bookDiv.innerHTML = `
+    <h2></h2>
+    <h3></h3>
+    <button type="button" onmousedown="removeBook(${bookId})">Remove</button>`;
 
-    bookList.appendChild(bookDiv);
-  });
-}
+  bookDiv.children[0].textContent = `${title}`;
+  bookDiv.children[1].textContent = `${author}`;
 
-function refreshBookList() {
-  bookList.innerHTML = '';
-  displayBooks();
+  bookList.appendChild(bookDiv);
 }
 
-function addBook() {
-  const currentTitle = bookForm.elements.title.value;
-  const currentAuthor = bookForm.elements.author.value;
+function displayStoredBooks() {
+  books.forEach((book, bookNum) => {
+    booksObj.createBook(book.title, book.author);
+    addBookToPage(book.title, book.author, bookNum);
+  });
+}
```

Comment:
this is also odd because you already have the books array in the object but you are adding to it again. because you are using the books array instead of the bookobj.books array

Reply:
Oh yes I see what you are saying